cmd/compose/pull.go

```diff
@@ -67,23 +68,29 @@ func pullCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	return cmd
 }
 
+func FilterServices(project *types.Project, services []string) error {
```

Comment:
Don't need to be exporte. Also name is bit confusing. Maybe `withSelectedServicesOnly` or something comparable?

Reply:
Done!